src/ui/wxWidgets/AddEditPropSheetDlg.cpp

```diff
@@ -3031,8 +3031,8 @@ void AddEditPropSheetDlg::OnEasyReadCBClick(wxCommandEvent& evt)
     // Check if pronounceable is also set - forbid both
     if (m_PasswordPolicyUsePronounceableCtrl->GetValue()) {
       m_PasswordPolicyUseEasyCtrl->SetValue(false);
-      wxMessageBox(_("Sorry, \"easy-to-read\" and \"pronounceable\" cannot be both selected"),
-                   _("Error"), wxOK|wxICON_ERROR, this);
+      wxMessageBox(_("\"Pronounceable\" and \"easy-to-read\" cannot be both selected."),
```

Comment:
Nit: the order of options has changed. I think the original intent was for both dialogs to say "<This option> and <other option> cannot be both selected".